builder/partials/header.hbs

```diff
@@ -14,7 +14,20 @@
         </a>
     </div>
 </header>
-
+<nav class="mdl-card mdl-shadow--2dp card">
+    <select onchange="window.location.href = event.target.value">
+        <option value="">All levels</option>
+        {{#each level}}
+            <option
+                {{#if (eq ../path (concat 'levels/' (concat this.urlSafe '.html') ) ) }}
```

Comment:
Lol, this looks very complicated, is there any way to simplify it?

Reply:
simplified